src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java

```diff
@@ -43,6 +45,119 @@ public String getSummary() {
         }
     }
 
+    /**
+     * Information about a package as returned by pkg.info_installed and
+     * pkg.info_available
+     */
+    public static class Info {
```

Comment:
Are you sure this should be named just `Info`? Wouldn't something like `PackageInfo` make it more clear what this is?